src/tools/searchDocs/searchIndex.ts

```diff
@@ -12,8 +11,7 @@ async function fetchAvailableVersions(
 > {
   try {
     const url = `${POWERTOOLS_BASE_URL}/${runtime}/versions.json`;
-    const response = await fetchService.fetch(url, {
-      contentType: ContentType.WEB_PAGE,
+    const response = await fetch(url, {
```

Comment:
Do we need a timeout here?

Reply:
Haven't looked at the stuff under `searchIndex` at all yet (will do in another PR), they're part of the diff because I moved things around - but probably yes.

For now I'll leave as it is.